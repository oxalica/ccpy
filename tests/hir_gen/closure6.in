def f():
  x = 1
  def g():
    nonlocal x
    global x
