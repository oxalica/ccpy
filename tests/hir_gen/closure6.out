Gen fail: Declare nonlocal name as global
